src/sections/dashboard/sidebar/VerticalSidebar.js

```diff
@@ -119,21 +121,37 @@ export default function VerticalSidebar() {
                       {/* Nested Children List */}
                       <List disablePadding>
                         {item.children.map((child) => (
-                          <ListItem key={child.id} disablePadding>
+                          <ListItem
+                            key={child.id}
+                            disablePadding
+                            sx={{
+                              ...(!isOpen && { padding: '8px 0' }),
+                            }}
+                          >
                             <ListItemButtonStyled
                               selected={pathname.includes(child.path)}
                               disableRipple
                               sx={{
-                                paddingLeft: '18px',
+                                padding: '0 16px 0 18px',
+                                display: 'flex',
+                                justifyContent: 'center',
+                                alignItems: 'flex-start',
                               }}
                               onClick={() => router.push(child.path)}
                             >
-                              <Box sx={{ display: 'flex' }}>
+                              <Box
+                                sx={{
+                                  ...(isOpen && {
```

Comment:
same [this](https://github.com/duchieupham/vietqr_web_2.0/pull/74/files#r1786606290)